examples/20190806_ThermalIR/run_example_modtran_one.py

```diff
@@ -1,11 +0,0 @@
-# David R Thompson, Adam Erickson, Jay Fahlen
```

Comment:
Moved to `isofit/test/test_examples.py`. I don't see a reason to maintain multiple versions of the configuration.